src/infinity-menu.js

```diff
@@ -113,12 +115,16 @@ export default class InfinityMenu extends React.Component {
 				return this.findFilted(p, c, k);
 			}, []) : [];
 			if (filteredSubFolder.length !== 0) {
-				node.isOpen = true;
+				node.isSearchOpen = true;
```

Comment:
because this part is only open in search and `isOpen` is open under the non-search